ragen/env/search/env.py

```
@@ -0,0 +1,550 @@
+        if self.config.search_type == "server":
+            return self._init_server_search()
+        else:
+            return self._init_real_search()
+    
+    
+    def _init_real_search(self):
+        """Initialize real search (Google, SerpAPI, etc.)."""
+        try:
+            from .real_search import get_real_search_adapter
+            return get_real_search_adapter(self.config)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)